deployments/deploy.ps1

```diff
@@ -42,9 +52,11 @@ try {
     $la2 = Get-PfLogAnalytics
     $la3 = Get-PfLogAnalytics -Name "Test" -ResourceGroup "Test"
   
-    $results = Deploy-PfDeploymentContext
-    $laDef = $results.GetByName($la.Options.Name)
+    $stateCollection = Deploy-PfDeploymentContext
+    $laDef = $stateCollection.GetResourceByName($la.Options.Name)
     $laDef
+    [hashtable]$ht = $kv.Options.Tags
+    $ht
 
```

Comment:
This is me testing the string dictionary to be converted back to hashtable and it works.